pom.xml

```diff
@@ -16,7 +16,7 @@
     <parent>
         <artifactId>che-parent</artifactId>
         <groupId>org.eclipse.che</groupId>
-        <version>5.19.0-SNAPSHOT</version>
+        <version>5.20.0-SNAPSHOT</version>
```

Comment:
You need to update https://github.com/redhat-developer/rh-che/blob/multi-tenant-rh-che/config#L23 too

Reply:
Fixed